app/javascript/App/bundles/common/components/CoinTags.tsx

```diff
@@ -1,21 +1,36 @@
 import * as React from 'react'
-import { ItemWithCoinLinkData } from '../types'
+import { ItemWithCoinLinkData, CoinLinkData, CoinClickHandler } from '../types'
 
 interface Props {
   itemWithCoinLinkData: ItemWithCoinLinkData
-  // selectCoin: (CoinLinkData) => void, // FIXME
+  selectCoin?: CoinClickHandler
+  getLink?: (coinData: CoinLinkData) => string
 }
 
-const CoinTags = ({ itemWithCoinLinkData /*selectCoin*/ }: Props) => (
+const CoinTags = ({ itemWithCoinLinkData, selectCoin, getLink }: Props) => (
   <div>
-    {itemWithCoinLinkData.coin_link_data.map((data, index) => (
-      <a
-        key={index}
-        className="tag pointer" /* onClick={() => selectCoin(data)}*/
-      >
-        {data.symbol}
-      </a>
-    ))}
+    {itemWithCoinLinkData.coin_link_data.map((data, index) => {
+      const isClickable = !!selectCoin
+      const onClickHandler = isClickable
```

Comment:
Would it be easier to just accept `onClick` as a prop? This way `CoinTags` do not need to know about selecting coins